pytest_rts/plugin.py

```diff
@@ -1,14 +1,24 @@
 """Code for pytest-rts plugin logic"""
 import logging
 import os
+import sqlite3
```

Comment:
PEP-8 says that import groups (standard library, third-party, local) should be separated by a blank line.